debian/octoscreen.service

```diff
@@ -11,7 +11,7 @@ ExecStart=/usr/bin/xinit /usr/bin/OctoScreen -- :0 -nolisten tcp -nocursor
 ExecStartPost=/bin/bash /etc/octoscreen/disablescreenblank.sh 0
 StandardOutput=journal
 Restart=always
-WatchdogSec=10s
+WatchdogSec=20s
```

Comment:
Nice catch!  BTW, this was something I was going to look into in 2.8.  I haven't been very communicative on what my plans are, so I posted a roadmap doc in the wiki (https://github.com/Z-Bolt/OctoScreen/wiki/Project-Roadmap)